tests/acceptance/test_med_safety_dsl.py

```diff
@@ -0,0 +1,289 @@
+from databuilder.tables import (
+    clinical_events,
+    hospital_admissions,
+    patient_demographics,
+    practice_registrations,
+)
+from tests.lib.tpp_schema import organisation, patient, registration, snomed_event
+
+from .codelists import gib_admissions_codelist, oral_nsaid_codelist, ppi_codelist
+
+EMERGENCY_ADMISSIONS_CODELIST = [
+    "21",  # Emergency Admission: Emergency Care Department or dental casualty department of the Health Care Provider
+    "22",  # Emergency Admission: GENERAL PRACTITIONER: after a request for immediate admission has been made direct to a Hospital Provider, i.e. not through a Bed bureau, by a GENERAL PRACTITIONER or deputy
+    "23",  # Emergency Admission: Bed bureau
+    "24",  # Emergency Admission: Consultant Clinic, of this or another Health Care Provider
+    "25",  # Emergency Admission: Admission via Mental Health Crisis Resolution Team
+    "2A",  # Emergency Admission: Emergency Care Department of another provider where the PATIENT  had not been admitted
+    "2B",  # Emergency Admission: Transfer of an admitted PATIENT from another Hospital Provider in an emergency
+    "2D",  # Emergency Admission: Other emergency admission
+    "28",  # Emergency Admission: Other means, examples are:
+    # - admitted from the Emergency Care Department of another provider where they had not been admitted
+    # - transfer of an admitted PATIENT from another Hospital Provider in an emergency
+    # - baby born at home as intended
+]
+
+
+# The v1 study definition uses a placeholder codelist, but for convenience we will use a
+# list.
+PLACEHOLDER_CODELIST = []
```

Comment:
Can you explain this "placeholder"? Why not a proper codelist?

Reply:
The v1 study definition references two placeholder codelists:

* `PLACEHOLDER_admissions_codelist`, which is a hard-coded list of codes
* `PLACEHOLDER_drugs_codelist`, which is [an OpenCodelists codelist][1]

I used an empty list as to represent "work in progress", as this seemed to be the intent of the v1 study definition. However, that's confusing 🙂, so I'll use the same hard-coded list of codes for the admissions codelist. I'll also use the same OpenCodelists codelist for the drugs codelist; we already reference this as `oral_nsaid_codelist`, so I'll prefer that name to `PLACEHOLDER_drugs_codelist`. See 41ca62c.

[1]: https://www.opencodelists.org/codelist/pincer/nsaid/23edd06e/

Reply:
There is already an event from `oral_nsaid_codelist`, which is why the tests failed. I have modified them to reflect the event and provided more information about why we are introducing redundancy. See 04f2442.